cmd/heighliner/github_repository.go

```diff
@@ -44,11 +45,22 @@ func ghpcCommand(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	period := 10 * time.Minute
```

Comment:
I think you can just set it up in the attributes `ReconciliationPeriod string 'long:"…" default:"10m"'`